src/main/cms/cms_menu_imu.c

```diff
@@ -319,9 +311,7 @@ static const OSD_Entry cmsx_menuSimplifiedTuningEntries[] =
     { "DTERM TUNING",   OME_TAB,   NULL, &(OSD_TAB_t)   { &cmsx_simplified_dterm_filter, 1, lookupTableOffOn }, 0 },
     { "DTERM MULT",     OME_FLOAT, NULL, &(OSD_FLOAT_t) { &cmsx_simplified_dterm_filter_multiplier, SIMPLIFIED_TUNING_MIN, SIMPLIFIED_TUNING_MAX, 1, 10 }, 0 },
 
-    { "-- GENERAL --", OME_Label, NULL, NULL, 0},
-    { "APPLY TUNING",  OME_Funcall, cmsx_applySimplifiedTuning, NULL, 0 },
-
+    { "--", OME_Label, NULL, NULL, 0},
     { "BACK", OME_Back, NULL, NULL, 0 },
```

Comment:
I think this should make it clear that this will apply the simplified settings, so it should probably be 'APPLY' or 'APPLY TUNING'.

Reply:
Yes, but the behavior is the same on other screens and they all use "back" label. Exiting the menu by "yawing" left has also the same effect. I would leave it the same just for the sake of consistency. Anyway no strong opinion, just opening discussion.

Reply:
I would not say that the behaviour is the same as on other screens - on other screens, 'back' only persists the settings that are edited on the screen, here the settings on the screen are persisted AND calculated values based on the new settings are applied to a lot of other parameters.
I think this should be indicated appropriately to the user.